src/partials/navbar-end.hbs

```diff
@@ -110,46 +57,46 @@
                 <li><a href="/docs/cypher-manual/current/functions/vector/" class="project-link">Vector search functions</a></li>
-                <li><a href="/docs/graphql/5/directives/indexes-and-constraints/#_vector_index_search" class="project-link">GraphQL vector index search documentation</a></li>
+                <li><a href="/docs/graphql/current/directives/indexes-and-constraints/#_vector_index_search" class="project-link">GraphQL vector index search</a></li>
               </ul>
             </div>
-            
+
             <div class="navbar-item project">
               <span class="project-name" href="/docs/create-applications">Create applications</span>
               <ul class="project-links">
-                <li><a href="{{#with (docs-driver-manual-url page 'python-manual')}}{{{this}}}{{/with}}" class="project-link">Python Driver</a></li>
-                <li><a href="{{#with (docs-driver-manual-url page 'go-manual')}}{{{this}}}{{/with}}" class="project-link">Go Driver</a></li>
-                <li><a href="{{#with (docs-driver-manual-url page 'java-manual')}}{{{this}}}{{/with}}" class="project-link">Java Driver</a></li>
-                <li><a href="{{#with (docs-driver-manual-url page 'jdbc-manual')}}{{{this}}}{{/with}}" class="project-link">JDBC Driver</a></li>
-                <li><a href="{{#with (docs-driver-manual-url page 'javascript-manual')}}{{{this}}}{{/with}}" class="project-link">JavaScript Driver</a></li>
-                <li><a href="{{#with (docs-driver-manual-url page 'dotnet-manual')}}{{{this}}}{{/with}}" class="project-link">.Net Driver</a></li>
-                <li><a href="/docs/graphql-manual/current/" class="project-link">Neo4j GraphQL Library</a></li>
-                <li><a href="/docs/nvl/current/" class="project-link">Neo4j Visualization Library</a></li>
+                <li><a href="{{#with (docs-driver-manual-url page 'python-manual')}}{{{this}}}{{/with}}" class="project-link">Python</a></li>
+                <li><a href="{{#with (docs-driver-manual-url page 'go-manual')}}{{{this}}}{{/with}}" class="project-link">Go</a></li>
+                <li><a href="{{#with (docs-driver-manual-url page 'java-manual')}}{{{this}}}{{/with}}" class="project-link">Java</a></li>
+                <li><a href="{{#with (docs-driver-manual-url page 'jdbc-manual')}}{{{this}}}{{/with}}" class="project-link">JDBC</a></li>
+                <li><a href="{{#with (docs-driver-manual-url page 'javascript-manual')}}{{{this}}}{{/with}}" class="project-link">JavaScript</a></li>
+                <li><a href="{{#with (docs-driver-manual-url page 'dotnet-manual')}}{{{this}}}{{/with}}" class="project-link">.Net</a></li>
+                <li><a href="/docs/graphql-manual/current/" class="project-link">GraphQL</a></li>
+                <li><a href="/docs/nvl/current/" class="project-link">Visualization Library</a></li>
```

Comment:
This is quite often referred to as *NVL* so if we omit _Neo4j_ it becomes just *VL* and that's not right.